setup.py

```diff
@@ -46,4 +47,12 @@ def read(fname):
         },
     test_suite='nose.collector',
     zip_safe=False,
+    data_files=[
+        ('socorro/external/postgresql/raw_sql/procs',
+            glob.glob('socorro/external/postgresql/raw_sql/procs/*.sql')),
+        ('socorro/external/postgresql/raw_sql/views',
+            glob.glob('socorro/external/postgresql/raw_sql/views/*.sql')),
+        ('socorro/external/postgresql/raw_sql/types',
+            glob.glob('socorro/external/postgresql/raw_sql/types/*.sql')),
+    ],
```

Comment:
This means we have to run `python setup.py sdist` in addition to `python setup.py install`. 

I brought it up in channel earlier this morning and we figured that adding the extra step for this one thing was not helpful.